docs/shared/angular-tutorial/07-share-code.md

```diff
@@ -60,7 +60,7 @@ If you're using [VS Code](https://code.visualstudio.com/) it may be necessary at
 **Now update `apps/api/src/app/app.service.ts` to import the interface:**
 
 ```typescript
-import { Injectable } from '@nestjs/common';
```

Comment:
This should actually come from `@nestjs/common` because this is for the NestJS Application. What is the issue you are experiencing?

Reply:
Sorry, you (and the example as it was) are right - closing the PR.